src/super_gradients/training/models/all_architectures.py

```diff
@@ -115,6 +115,8 @@
                  "vit_base": vit_base,
                  "vit_large": vit_large,
                  "vit_huge": vit_huge,
+                 "beit_base_patch16_224": beit_base_patch16_224,
+                 "beit_large_patch16_224": beit_large_patch16_224
```

Comment:
we should have some consistency with ViTs. if we say "vit_base" and not "vit_base_patchX_X" then beit should be similar or the ViTs should be changed

Reply:
Agree, Ill change the Vitt's strings for now.